controller/volunteer_controller.py

```diff
@@ -0,0 +1,69 @@
+from models.volunteer import Volunteer
+from models.camp import Camp
+
+
+###---- Manage Volunteer Menu ----
+def create_volunteer(username: str,
+                     password: str,
+                     firstname: str,
+                     lastname: str,
+                     phone: str,
+                     camp: Camp) -> Volunteer:
+    """
+    Yunsy, Yingbo
+    """
+    pass
+
+
+def find_volunteer(username: str) -> Volunteer:
+    pass
+
+
+def view_volunteer_profile(volunteer: Volunteer) -> str:
+    """
+    Admin
```

Comment:
Does this mean that only admin can view? Needs minor clarification to avoid misunderstanding.